fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/cpp/utils/Attributes.xtend

```diff
@@ -213,6 +226,7 @@ class Attributes {
 		//	Attribute «attribute.name» class definition
 		class «attribute.name»Attrib: public Tango::«attribute.inheritance»
 		{
+		«attribute.attributeConstant»
 		public:
```

Comment:
1. This is not solving #74. The user wanted to have access to these constants (*"to allow the programmer to check the size of the spectrum"*). Currently they are private.
2. This will be not indented correctly in the generated code.
3. I also suggest to either put them at namespace level or drop attribute name prefix from constant names. Currently it is repeated, e.g. for 'myimg' attribute, this name appears twice as we have: `myimgAttrib::myimg_X_DATA_SIZE`. I think `myimgAttrib::X_DATA_SIZE` would be enough.

Reply:
Update the attribute definition and now we have:
```cpp
//	Attribute normalImageTest class definition
class normalImageTestAttrib: public Tango::ImageAttr
{
public:
    // Constants for normalImageTest attribute
    constexpr static long X_DATA_SIZE = 22;
    constexpr static long Y_DATA_SIZE = 54;
	normalImageTestAttrib():ImageAttr("normalImageTest",
			Tango::DEV_DOUBLE, Tango::READ, normalImageTestAttrib::X_DATA_SIZE, normalImageTestAttrib::Y_DATA_SIZE) {};
	~normalImageTestAttrib() {};
	virtual void read(Tango::DeviceImpl *dev,Tango::Attribute &att)
		{(static_cast<Test *>(dev))->read_normalImageTest(att);}
	virtual bool is_allowed(Tango::DeviceImpl *dev,Tango::AttReqType ty)
		{return (static_cast<Test *>(dev))->is_normalImageTest_allowed(ty);}
};
```

- now the constants are public
- added namespace to the attrribute uses

Reply:
Please fix indentation to be consistent with the rest of this class. Don't mix spaces and tabs.

Reply:
Sure, thanks for noticing that. Done :)